### PR TITLE
Switch to default org issue templates

### PR DESCRIPTION
**Pull request by adityaoberai**
## What does this PR do?

Adds newer Appwrite versions to the Bug issue template

## Test Plan

image

## Related PRs and Issues

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -37,6 +37,15 @@ body:
...
       options:
+        - Version 1.3.x
+        - Version 1.2.x
+        - Version 1.1.x
+        - Version 1.0.x      
+        - Version 0.15.x
+        - Version 0.14.x
+        - Version 0.13.x
+        - Version 0.12.x
+        - Version 0.11.x
```
Comment (stnguyen90) [inline .github/ISSUE_TEMPLATE/bug.yaml:48]: This file looks pretty generic. Can we maybe delete this file from this repo and have it use the default from https://github.com/appwrite/.github/blob/main/.github/ISSUE_TEMPLATE/bug.yaml?

Comment (adityaoberai) [reply .github/ISSUE_TEMPLATE/bug.yaml:48]: Sounds like a good idea
Should we remove these altogether from here, then?

Comment (adityaoberai) [reply .github/ISSUE_TEMPLATE/bug.yaml:48]: @stnguyen90 is that a good next step for now?

Comment (adityaoberai) [reply .github/ISSUE_TEMPLATE/bug.yaml:48]: @stnguyen90 lemme know whatever you think is a good next step here

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**